stdlib/UUIDs/src/UUIDs.jl

```diff
@@ -53,7 +55,7 @@ UUID("cfc395e8-590f-11e8-1f13-43a2532b2fa8")
 !!! compat "Julia 1.6"
     The default rng has been changed to use Random.RandomDevice as of Julia 1.6
 """
-function uuid1(rng::AbstractRNG=Random.RandomDevice())
+function uuid1(rng::AbstractRNG=uuid_rng)
```

Comment:
Why did you make this change? Creating a new `RandomDevice()` is extremely cheap on non-Windows, and on Windows having a global `uuid_rng` is not thread-safe...

Reply:
@rfourquet I guess misunderstood @JeffBezanson 's comment above?

> How about using a dedicated default RNG for UUIDs that's initialized from RandomDevice at startup?

Reply:
My interpretation of it was using an RNG like `default_rng()` (currently `MersenneTwister()` which is thread-safely initialized), which is therefore immune against the unqualified `Random.seed!()` but way faster than `RandomDevice()`, but "_initialized_ from `RandomDevice()`" means it's _seeded_ from `RandomDevice()` at startup.

Reply:
But Stefan's last comment suggests to keep it how it was, because the poor performance of `RandomDevice` might considered as a non-problem until someone complains.

Reply:
Okay! I reverted 507b454 . I'll leave it alone for now until there's something I should explicitly add/change/remove.